app.js

```diff
@@ -1,18 +1,78 @@
 require('dotenv').config();
-
+const path = require("path");
+const axios = require("axios");
 const express = require('express');
 const hbs = require('hbs');
+require("dotenv").config(); 
 
-// require spotify-web-api-node package here:
+hbs.registerPartials(__dirname + "/views/partials");
 
+// require spotify-web-api-node package here:
+const SpotifyWebApi = require('spotify-web-api-node');
 const app = express();
 
 app.set('view engine', 'hbs');
 app.set('views', __dirname + '/views');
 app.use(express.static(__dirname + '/public'));
 
 // setting the spotify-api goes here:
-
+const spotifyApi = new SpotifyWebApi({
+    clientId: process.env.CLIENT_ID,
+    clientSecret: process.env.CLIENT_SECRET
+  });
+  
+  // Retrieve an access token
+  spotifyApi
+    .clientCredentialsGrant()
+    .then(data => spotifyApi.setAccessToken(data.body['access_token']))
+    .catch(error => console.log('Something went wrong when retrieving an access token', error));
 // Our routes go here:
 
-app.listen(3000, () => console.log('My Spotify project running on port 3000 🎧 🥁 🎸 🔊'));
+app.get("/", (req, res) => {res.render("index")});
+
+app.get("/artist-search", (req, res) => {
+  const { artistName } = req.query;
+
+   spotifyApi
+    .searchArtists(artistName)
+    .then((data) => {
+      console.log("The received data from the API: ", data.body.artists.items);
```

Comment:
Remember to remove unnecessary `console.logs` after you're done building your application